README.md

```diff
@@ -55,10 +55,15 @@ Make sure you install rust from the main rust website. Cargo should take care of
 
 These are only needed if you're going to build a WebAssembly binary:
 ```bash
-$ sudo apt install binaryen
-$ cargo install -f wasm-bindgen-cli --version 0.2.100
+$ cargo install -f wasm-bindgen-cli --version 0.2.97
 $ cargo install basic-http-server
 $ rustup target add wasm32-unknown-unknown
+
+# binaryen version >= 110 required for a bug fix for wasm-opt
+# Refer to https://github.com/emilk/egui/pull/6848
+$ wget -P ~/ https://github.com/WebAssembly/binaryen/releases/download/version_123/binaryen-version_123-x86_64-linux.tar.gz
+$ tar xf ~/binaryen-version_123-x86_64-linux.tar.gz -C ~/
+$ export PATH=$PATH:~/binaryen-version_123/bin
```

Comment:
[870e687](https://github.com/open-rmf/rmf_site/pull/280/commits/870e6871434fb5245d4a8691800c9321eac51402): `binaryen` v110 or higher is required due to a bug that results in the web app crashing.
- Ubuntu 24.04 ships with v108, Ubuntu 25.04 ships with v120
- Our previous workaround for this was to fix `wasm-bindgen` at 0.2.93: https://github.com/open-rmf/rmf_site/issues/251
- However, now that we're moving to `bevy_egui` 0.33, we require at least `wasm-bindgen` 0.2.97
- The new workaround involves manually installing a working version of `binaryen`
- CI updated to use v123 directly as well